shopinvader_price_per_qty/README.rst

```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============================
+ShopInvader Price per Quantity
+==============================
+
+Thsi module extract the price per quantity in order to make in available in your shopinvader website
```

Comment:
can you elaborate a bit on this? It should mention pricelist etc.

Also, please use readme fragments.